packages/fast-development-site-react/src/components/shell/index.tsx

```diff
@@ -28,6 +28,7 @@ export enum ShellSlot {
 const style: ComponentStyles<IShellManagedClasses, IDevSiteDesignSystem> = {
     shell: {
         fontFamily: "Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif",
+        fontSize: "14px",
         height: "100vh"
```

Comment:
FYI - I think we'll need to update this as it currently a broken experience. 

A design specific question we'll want to land (cc: @Falkicon ) is what happens to this when dev tools are open or my browser screen is too short? Do we want a min-height here?